src/vcpkg/base/system.process.cpp

```diff
@@ -661,8 +661,9 @@ namespace vcpkg
         {
             auto pos = env_var.find('=');
             auto key = env_var.substr(0, pos);
-            if (Util::Sets::contains(env_strings, key) ||
-                Util::any_of(env_prefix_string, [&](auto&& group) { return Strings::starts_with(key, group); }))
+            if (Util::Sets::contains(env_strings, key) || Util::any_of(env_prefix_string, [&](auto&& group) {
```

Comment:
Is `Util::Sets::contains` case-insensitive? `SystemRoot` is in `env_strings`

Reply:
Good catch. It too late for me 😅 Will fix that :)